includes/models/model.llms.order.php

```diff
@@ -1703,6 +1743,80 @@ public function set_status( $status ) {
 
 	}
 
+	/**
+	 * Sets user-related metadata for the order.
+	 *
+	 * @since [version]
+	 *
+	 * @param array|LLMS_Student|WP_User|integer $user_or_data Accepts a raw array user meta-data or
+	 *                                                         an input string accepted by `llms_get_student()`.
+	 *                                                         When passing an existing user the data will be pulled
+	 *                                                         from the user metadata and saved to the order.
+	 * @return array {
+	 *     Returns an associative array representing the user metadata that was stored on the order.
+	 *
+	 *     @type integer $user_id            User's WP_User id.
```

Comment:
```suggestion
	 *     @type integer $user_id            User's WP_User ID.
```
The abbreviation for "identity" is ID. 
"id" means "the one of the three divisions of the psyche in psychoanalytic theory that is completely unconscious and is the source of psychic energy derived from instinctual needs and drives".
https://www.merriam-webster.com/dictionary/id